codespell_lib/data/dictionary.txt

```diff
@@ -5342,26 +5374,46 @@ notasion->notation
 noteable->notable
 noteably->notably
 noteriety->notoriety
+notfication->notification
+notfications->notifications
 noth->north
 nothern->northern
 noticable->noticeable
 noticably->noticeably
+notications->notifications
 noticeing->noticing
 noticible->noticeable
 notifcation->notification
+notifcations->notifications
+notifed->notified
+notifiy->notify
+notifiying->notifying
 nott->not
 notwhithstanding->notwithstanding
+not either->neither
+not quit->not quite
 noveau->nouveau
 Novermber->November
 nowdays->nowadays
 nowe->now
+no compliant->noncompliant
```

Comment:
this can be used legitimately

Reply:
True. These are good examples of #255 
I wonder if there is a way we can consider entering them in to codespell somehow and use some regex look aheads to utilize them?

Reply:
Not in this PR

Reply:
fair enough